cmd/scheduled-feed/main.go

```diff
@@ -45,33 +79,15 @@ func (handler *FeedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		b, err := json.Marshal(pkg)
 		if err != nil {
 			log.Printf("error marshaling package: %#v", pkg)
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			return
+			return processed, err
```

Comment:
I think that failing to marshal a pkg should be a soft failure, we can continue to publish the next package, with processed++ only happening on a successful publish. 

Obviously a failed publish isn't worth retrying as it is likely a problem with the publisher configuration or endpoint.

Reply:
I don't disagree, but this is the original behaviour as such I didn't want to make the change in this PR. I think adding a retry on the publisher attempt is worthwhile, but as you say hard failing the entire package list due to one marshal error seems OTT.

Reply:
Opened https://github.com/ossf/package-feeds/issues/100 and https://github.com/ossf/package-feeds/issues/101 to track these issues.